### ISSUE TITLE
Block gas limit issue

### ISSUE THREAD

**Issue by joeykrug** [ethereum/cpp-ethereum#1449]
Was block gas limit on alethzero raised to pi M for private chains too?
Gas limit seems to only be 1M

Getting this upon creating a new contract that takes 1.8M gas:
```Users/administrator/slave/build-cpp-osx-develop/build/libethereum/Executive.cpp(94): Throw in function bool dev::eth::Executive::setup()
Dynamic exception type: boost::exception_detail::clone_impl<dev::eth::BlockGasLimitReached>
std::exception::what: std::exception
[dev::tag_required*] = 985444
[dev::tag_got*] = 1832236```

**Comment by gavofyork** [comment]
can't see any immediate sign of this... could you give full reproduction instructions from a clean AZ install?

**Comment by joeykrug** [comment]
I just installed the latest OS X binary (this was a few days ago) and chain gas limit per block was x/1000000.

I installed the latest Windows binary just now and the gas limit is pi million.  I'm travelling for the next week and only have my windows/ubuntu machine.  

Either the issue has been fixed or it's an OS X only issue.

I'm going to close this issue, check OS X again when I get home, and only reopen if it's still 1M on OS X.
